plugin/pkg/admission/runtimeclass/admission.go

```diff
@@ -171,13 +181,43 @@ func setOverhead(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta1.Run
 	return nil
 }
 
-func validateOverhead(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta1.RuntimeClass) (err error) {
+func setScheduling(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta1.RuntimeClass) (err error) {
```

Comment:
nit: don't use a named return variable
```suggestion
func setScheduling(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta1.RuntimeClass) error {
```

Reply:
This whole file uses a named return variable. If this is necessary, I could do it in a follow-up PR, WDYT?

Reply:
sgtm